JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -15,479 +15,456 @@
+          omega       = omega, 
+          itemRestCor = itemRestCor, 
+          mu          = mu, 
+          sd          = sd
+        )
+      } 
+  } else {
+    variablesTemp <- variables
+    
+    if (is.null(variables)) 
+      variablesTemp <- "..."
+    
+    for (var in variablesTemp) {
+      results[[var]] <- list(
+        case        = var, 
+        alpha       = ".", 
+        lambda      = ".", 
+        omega       = ".", 
+        itemRestCor = ".", 
+        mu          = ".", 
+        sd          = "."
+      )
+    }
+  }
+  jaspResults[["itemContainer"]][["table"]]$addRows(results)
+}
 
-	return(table)
+# Output functions ----
+.reliabilityScaleTable <- function(jaspResults, dataset, options, ready) {
+  
+  if (!is.null(jaspResults[["scaleTable"]])) return()
+  
+  # Create table
+  scaleTable <- createJaspTable(title = "Scale Reliability Statistics")
+  dependList <- c("variables", "confAlpha", "mcDonaldScale", "alphaScale", "meanScale",
+                  "alphaScaleStandardized", "gutmannScale", "glbScale", "confAlpha",
+                  "averageInterItemCor",  "sdScale", "missingValues", "confAlphaLevel")
+  scaleTable$dependOn(dependList)
+  scaleTable$showSpecifiedColumnsOnly <- TRUE
+  scaleTable$position <- 1
+  
+  # Add columns to table
+  scaleTable$addColumnInfo(name = "case", title = "", type = "string")
+  if (options$meanScale)
+    scaleTable$addColumnInfo(name = "mu", title = "mean", type = "number")
+  if (options$sdScale)
+    scaleTable$addColumnInfo(name = "sd", title = "sd", type = "number")
+  if (options$mcDonaldScale)
+    scaleTable$addColumnInfo(name = "omega", title = "McDonald's \u03C9",     
+                             type = "number")
+  if (options$alphaScale)
+    scaleTable$addColumnInfo(name = "alpha", title = "Cronbach's \u03B1",
+                             type = "number")
+  if (options$gutmannScale)
+    scaleTable$addColumnInfo(name = "lambda", title = "Gutmann's \u03BB6",
+                             type = "number")
+  if (options$glbScale)
+    scaleTable$addColumnInfo(name = "glb", title = "Greatest lower bound",
+                             type = "number")
+  if (options$averageInterItemCor)
+    scaleTable$addColumnInfo(name = "rho", title = "Average interitem correlation",
+                             type = "number")
+  if (options$confAlpha){
+    overTitle <- paste0(100*options$confAlphaLevel, "% Confidence Interval")
+    scaleTable$addColumnInfo(name = "lower", title = "Lower", type = "number", 
+                             overtitle = overTitle)
+    scaleTable$addColumnInfo(name = "upper", title = "Upper", type = "number", 
+                             overtitle = overTitle)
+  }
+  
+  if (options$missingValues == "excludeCasesListwise") 
+    exclwise = " listwise"
+  else 
+    exclwise = " pairwise"
+  
+  variables <- unlist(options$variables)
+  nObs      <- nrow(dataset)
+  nExcluded <- sum(!complete.cases(dataset))
```

Comment:
`dataset` can be `NULL`, in which case you get an error. You should probably have already returned at this point by checking `ready`